Cubical/Algebra/CommMonoid/CommMonoidProd.agda

```diff
@@ -0,0 +1,37 @@
+{-# OPTIONS --safe #-}
+module Cubical.Algebra.CommMonoid.CommMonoidProd where
+
+open import Cubical.Foundations.HLevels
+
+open import Cubical.Data.Sigma
+
+open import Cubical.Algebra.Monoid
+open import Cubical.Algebra.Semigroup
+open import Cubical.Algebra.CommMonoid.Base
+
+
+open CommMonoidStr
+open IsCommMonoid hiding (rid ; lid)
+open IsMonoid hiding (rid ; lid)
+open IsSemigroup
+
+
+private
+  variable
+    ℓ ℓ' : Level
+
+
+CommMonoidProd : CommMonoid ℓ → CommMonoid ℓ' → CommMonoid (ℓ-max ℓ ℓ')
+CommMonoidProd M N = makeCommMonoid ε× _·×_ is-set× assoc× rid× comm×
+  where
+  ε× = (ε (snd M)) , (ε (snd N))
+  
+  _·×_ = λ x y → _·_ (snd M) (fst x) (fst y) , _·_ (snd N) (snd x) (snd y)
```

Comment:
Here and in the following lines, it might be better to give type signatures and remove the 'λ's

Reply:
Done!